pkgs/os-specific/linux/tpacpi-bat/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, perl, kmod, acpi_call, coreutils }:
```

Comment:
`acpi_call` & `coreutils` are not used below. Just drop them.

The `acpi_call`  module is a runtime dependency but that has to be specified in the NixOS configuration.